var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -120,6 +120,7 @@ def configure_args(self):
         else:
             args.append("--with-kdreg=no")
 
+        args.append("--enable-only")
```

Comment:
Was there a particular reason why this was necessary? Libfabric's configure script is reasonable and only builds providers that have everything it requires in the build environment.

Reply:
I needed to add this when building on my laptop, with only the default fabrics enabled. However, I just tried again, and was unable to reproduce the error. Perhaps I could remove this change.

Reply:
Yeah, let's drop this.